src/dispatching/repls.rs

```diff
@@ -1,4 +1,11 @@
-//! REPLs for dispatching updates.
+//! [REPL]s for dispatching updates.
+//!
+//! This module provides utilities for easy update handling. They accept a
+//! single "handler" function that processes all updates of a certain kind. Note
+//! that REPLs are meant to be used for simple scenarios, such as prototyping,
+//! inasmuch they lack configuration and some advanced features.
```

Comment:
Link "advanced features" to the "Dispatching or REPLs?" section from `src/dispatching.rs`.